src/components/SendTx.vue

```diff
@@ -200,6 +200,45 @@ enum RecipientType {
         @Prop({type: String, default: ''}) public message!: string;
         @Prop({type: Number, default: 0}) public validityStartHeight!: number;
 
+        public $refs: {
+            accountDetails?: AccountDetails,
+            amountWithFee?: AmountWithFee,
+            messageInput?: LabelInput,
+            feeSetter?: SelectBar,
+            address?: AddressInput,
+        };
+
+        public focus(dontFocusOnMobile = false) {
```

Comment:
I think the default here should be the user-friendly option: `dontFocusOnMobile = true`. That would also remove the need to pass `true` in all focus() calls in this PR.